workflows/smrnaseq.nf

```diff
@@ -0,0 +1,241 @@
+/*
+========================================================================================
+    VALIDATE INPUTS
+========================================================================================
+*/
+
+def summary_params = NfcoreSchema.paramsSummaryMap(workflow, params)
+
+// Validate input parameters
+WorkflowSmrnaseq.initialise(params, log)
```

Comment:
Now the validation of the parameters is not included in the library file, see my comment on the corresponding file